.github/workflows/release.yml

```diff
@@ -30,30 +27,34 @@ jobs:
         run: npm ci
       - name: Release
         run: |
-          readonly COMMIT_MESSAGE="${{ github.event.commits[0].message }}"
-          if [[ "${COMMIT_MESSAGE: -11: -1}" == 'ci release' ]]; then
-            echo "::set-env name=NEXT_VERSION::$(npm run release:version --silent)"
-            npx --no-install standard-version
-          else
-            readonly RELEASE_TYPE="${COMMIT_MESSAGE: -6: -1}"
-            echo "::set-env name=NEXT_VERSION::$(
-                npm run release:version --silent -- --releaseAs "${RELEASE_TYPE}"
-                )"
-            npx --no-install standard-version --release-as "${RELEASE_TYPE}"
+          SEMVER_REGEX="v(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)"
+          if [[ ! "${{ github.event.commits[0].message }}" =~ ${SEMVER_REGEX} ]]; then
+            echo Not a semver like version - aborting: ${1}
+            exit 1
           fi
+          NEXT_VERSION="${BASH_REMATCH[1]}.${BASH_REMATCH[2]}.${BASH_REMATCH[3]}"
+          MAJOR_VERSION="${BASH_REMATCH[1]}"
+          "::set-env name=NEXT_VERSION::${NEXT_VERSION}"
```

Comment:
This would need to start with `echo` https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable